migration_challenge_keras_image/Local Notebook.ipynb

```diff
@@ -138,39 +138,67 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%%time\n",
+    "%time\n",
```

Comment:
Need the second `%` here to use the cell timer magic rather than the line timer magic. E.g:

```python
%time sleep(5)  # 5sec
```

```python
%%time  # 7sec
sleep(5)
sleep(2)
```

```python
%time  # 0sec (misleading)
sleep(5)
sleep(2)
```

Reply:
Fixed in 89dd6c575b8c787d96ece5a57d51fd36193d2e42